lib/EnsEMBL/REST/Controller/Info.pm

```diff
@@ -155,6 +155,74 @@ sub biotypes_GET :Local :Args(1) {
   return;
 }
 
+sub biotype_groups_GET { }
+
+sub biotype_groups : Chained('/') PathPart("info/biotypes/groups") Args(0) ActionClass('REST') {
+  my ($self, $c) = @_;
+
+  my $groups;
+
+  try {
+    $groups = $c->model('Biotype')->fetch_biotype_groups;
+
+  } catch {
+    $c->go('ReturnError', 'from_ensembl', [qq{$_}]) if $_ =~ /STACK/;
+    $c->go('ReturnError', 'custom', [qq{$_}]);
+  };
+
+  $self->status_ok($c, entity => $groups);
+  return;
+}
+
+
+sub biotype_group_GET { }
+
+sub biotype_group : Chained('/') PathPart("info/biotypes/groups") Args(1) ActionClass('REST') {
+  my ($self, $c, $group) = @_;
+
+  my $biotypes;
+
+  try {
+    $biotypes = $c->model('Biotype')->fetch_biotypes_by_group($group);
+
+  } catch {
+    $c->go('ReturnError', 'from_ensembl', [qq{$_}]) if $_ =~ /STACK/;
+    $c->go('ReturnError', 'custom', [qq{$_}]);
+  };
+
+  $self->status_ok($c, entity => $biotypes);
+  return;
+}
+
+sub biotype_names_GET { }
+
+sub biotype_names : Chained('/') PathPart("info/biotypes/name") Args(0) ActionClass('REST') {
```

Comment:
Do I understand correctly that you've made an entire route for detecting a missing parameter?

Reply:
yes...
without that route it goes straight into the old info/biotypes/:species endpoint, complaining the species 'name' does not exist.
I can however make the check there and remove that useless route.